presto-main/src/main/java/io/prestosql/server/Server.java

```diff
@@ -55,20 +55,18 @@
 import java.nio.file.Paths;
 import java.util.Set;
 
-import static com.google.common.base.MoreObjects.firstNonNull;
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
 import static io.airlift.discovery.client.ServiceAnnouncement.ServiceAnnouncementBuilder;
 import static io.airlift.discovery.client.ServiceAnnouncement.serviceAnnouncement;
 import static io.prestosql.server.PrestoSystemRequirements.verifyJvmRequirements;
 import static io.prestosql.server.PrestoSystemRequirements.verifySystemTimeIsReasonable;
 import static java.nio.file.LinkOption.NOFOLLOW_LINKS;
 
-public class PrestoServer
+public class Server
 {
     public static void main(String[] args)
     {
-        String version = PrestoServer.class.getPackage().getImplementationVersion();
-        new PrestoServer().start(firstNonNull(version, "unknown"));
+        new Server().start("testversion");
```

Comment:
Add a comment that this `main` method is for developers' only.

In fact, maybe we remove this method at all?
We could change README -- instead of changing the class name, you could change the module name to use the new module.

Reply:
We'd have to move the `etc` directory as well, but that seems better. I was thinking this would be a temporary change, but it seems good to keep this special main method around to provide a good error message when we update the JVM requirement.